app/views/user_mailer/monthly_owner_wrapup.html.slim

```diff
@@ -0,0 +1,44 @@
+p Hello #{@owner.display_name},
```

Comment:
Needs the logo at the top.

Reply:
It should be inheriting the logo from the mailer layout.  (I don't want to send out a live test, since my database is a copy of production.)

Reply:
I noticed the 100% complete one had a (broken) logo image at the top, but the monthly one didn't...  
at least in the screenshots you sent.

Reply:
They inherit exactly the same way, though.  I don't actually know if they're broken, but they look the same on my actual screen. (It might have been scrolled up for the screenshot.)